.github/workflows/ci.yml

```diff
@@ -102,6 +102,11 @@ jobs:
         run: |
           docker run --rm "$CORE_CI_IMAGE" bash -c "cd /opt/npm_and_yarn && npm run lint"
           docker run --rm "$CORE_CI_IMAGE" bash -c "cd /opt/npm_and_yarn && npm test"
+      - name: Run bundler v1 native helper specs
+        if: matrix.suite == 'bundler'
```

Comment:
Not sure if it's a big deal right for this PR, but I guess we should add `bundler2` to the matrix rather than mess with this existing key?

Reply:
Ah yes, was also thinking we could run them in in the same suite a second step as I think they run quickly. Each matrix ends up building the entire image as it's set up currently.

Reply:
Good call, that makes sense!